x/launch/types/errors.go

```diff
@@ -38,4 +38,5 @@ var (
 	ErrChainHasCampaign            = sdkerrors.Register(ModuleName, 29, "chain already is associated with a campaign")
 	ErrAddChainToCampaign          = sdkerrors.Register(ModuleName, 30, "unable to add chain to campaign")
 	ErrRemainingTimeNotPositive    = sdkerrors.Register(ModuleName, 31, "invalid remaining time")
+	ErrChainIBCConnected           = sdkerrors.Register(ModuleName, 32, "chain is already IBC connected")
```

Comment:
Same comment as above for the error name. Should specify `monitoring` is concerned